supreme_court_predictions/processing/token_aggregation.py

```diff
@@ -40,6 +44,50 @@ def __init__(self, debug_mode=False):
         self.utterances = self.get_utterances()
         self.utterance_sides = self.append_side(self.utterances)
 
+    @staticmethod
+    def get_sentence_count(text):
+        """
+        Calculates the number of sentences and the number of words per sentence
```

Comment:
You're missing the input param part of it tho. 😱

Reply:
i am DUM michael pls